src/styles/Book.module.css

```diff
@@ -0,0 +1,109 @@
+.bookcard {
```

Comment:
- Kindly add space to the top, left and right of the book section.
<img width="1439" alt="Screenshot 2023-03-18 at 1 57 47 PM" src="https://user-images.githubusercontent.com/25887253/226101955-98bf13cd-1c90-4a36-ab7e-b894da1c8a8e.png">

- Kindly add space between the books. Each card has a space under it like on the zeplin design.
<img width="657" alt="Screenshot 2023-03-18 at 1 59 30 PM" src="https://user-images.githubusercontent.com/25887253/226102041-a0942583-26d5-47d9-8785-0360622eb308.png">